### PR TITLE
Flesh out start script for Whiteblock docker image.

### PR DESCRIPTION
**Pull request by ajsutton**
## PR Description
Implements the start script for Whiteblock so it actually starts Artemis.

### CODE REVIEW

File: scripts/configurator.sh [hunk 1]
```diff
@@ -1,11 +1,11 @@
-#!/bin/sh
+#!/bin/bash
```
Comment (ajsutton) [inline scripts/configurator.sh]: `if [[ ... ]]` syntax is not supported by `sh` (sh on OS X is just bash but it's actually sh on Debian & Ubuntu).

File: scripts/configurator.sh [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 if [[ "$#" -ne 3 ]]
 then 
   echo "Usage: sh configurator.sh CONFIG_PATH VARIABLE_NAME VARIABLE_VALUE"
   exit 1
 fi
 
 
-sed -i.bak "s/$2\ =.*/$2\ =\ $3/g" $1
+sed -i.bak "s!#*.*$2\ =.*!$2\ =\ $3!g" $1
```
Comment (ajsutton) [inline scripts/configurator.sh]: Changed the delimiter char so that we can use this to insert paths (which contain / chars) into the config.

**Comment by jrhea** [comment]
I haven't forgot about this.  i need to make sure that my scripts still work with these changes.

**Review by jrhea — APPROVED** [review]
lgtm

**Pull request merged**